boards/cc1312-launchpad/Makefile.features

```diff
@@ -0,0 +1,7 @@
+CPU = cc13x2
+CPU_MODEL = cc1312r1f3
+
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio
```

Comment:
No `periph_gpio_irq` ?
Support is provided by the CPU:
https://github.com/RIOT-OS/RIOT/blob/0703449c0f58c082ed9dbc0253b89cc6e7ba2992/cpu/cc26xx_cc13xx/periph/gpio.c#L90-L92